script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -892,7 +892,7 @@ Which means... now it's time for me to hold up my end of the bargain.
 -- Page 495, Offset 15989.
 -- いいよ、俺も吸血鬼の排除には賛成だし。怖いのはイヤだけど、我慢して協力してやる」
 Why not. I fully endorse getting rid of the vampires anyway.
- It sounds terrifying, and I don't like that, but I'll bear with it and cooperate."
+ It sounds terrifying, and I don't like it, but I'll bear with it and cooperate."
```

Comment:
He's actually saying he doesn't like being afraid here, pretty sure - 怖いのはイヤだけど hence "I don't like that [it sounds terrifying" instead of "I don't like it [getting rid of the vampires]". Should probably just make it more clear - 
> It sounds terrifying, so I'm not looking forward to it, but...